api/resource.rb

```diff
@@ -46,6 +45,8 @@ module Properties
 
     include Properties
 
+    # Properties can be overridden via Provider::PropertyOverride
+    attr_reader :properties
     attr_reader :__product
```

Comment:
Add line between :properties and :__product to scope the comment.

Reply:
Done